benefits/core/views.py

```diff
@@ -33,22 +33,18 @@ def index(request):
         agency = agencies[0]
         return redirect(agency.index_url)
```

Comment:
I'm thinking this whole check/redirect can go away now that we have a proper selection experience (and 2 agencies)

Reply:
Done in 069991d815e846f13e31a266a52e9a2c17eb71d7

Reply:
Had to rebase to fix a test, so the new hash that this was done in is 4719906ddaa9832aea249d0d70151c8a72fc10eb